### ISSUE TITLE
`UI.create_cursor` documentation: logical cursor size and incorrectly implies coordinates begin at 1

### ISSUE THREAD

**Issue by DanRathbun** [SketchUp/api-issue-tracker#918]
#### SketchUp Ruby API Documentation Issue

**`UI.create_cursor` documentation: logical cursor size and incorrectly implies coordinates begin at 1**

The [`UI::create_cursor`](https://ruby.sketchup.com/UI.html#create_cursor-class_method) documentation is not at all clear **what** that the **nominal** cursor size is.

... and what happens with respect to display scaling. (Is it automatic, or is [`UI::scale_factor`](https://ruby.sketchup.com/UI.html#scale_factor-class_method) required to be used?)

Also, the explanation for the `hot_y` parameter implies coordinates begin at 1 instead of zero. It says ...
> * **`hot_y`** (`Integer`) — A y coordinate that is the “hotpoint” for the cursor computed from the top edge of the of your cursor image. For example, a value of `(hot_x, hot_y) = (5,10)` would identify the hotpoint of the cursor at 5 pixels from the left edge of your cursor image and 10 pixels from the top edge of your cursor image.

When it should conclude ...

 "a value of `(hot_x, hot_y) = (5,10)` would identify the hotpoint of the cursor at 6 pixels from the left edge of your cursor image and 11 pixels from the top edge of your cursor image."

Ie, a `(0,0)` parameter list means the top left corner of the cursor bounds.

**Comment by Fredosixx** [comment]
I agree with Dan that the doc is not clear and that it seems that the hot spot is based on a nominal size 24x24, not on the actual size.

This is annoying, but **a fix of this problem should be clearly notified to script developers**, because we have workarounds in place.

**Comment by J-E-L** [comment]
**UI.create_cursor(cursorPath, dx, dy)  has different behaviors**

Windows: SketchUp Version 2020 - 2023:
dx , dy are Integer from 0 to 31!

OS X: SketchUp Version 2020 - 2022:
dx , dy are Integer from 0 to 31!

OS X: SketchUp Version 2023:
dx , dy are Integer from 0 to 23!

If dx or dy is greater than max, the cursor will not be displayed.

All my cursors with dx or dy greater than 23 that worked until version 2022 no longer work in 2023.

I have now done a workaround for Sketchup 2023 in "3skeng"
